### PR TITLE
feat(all): store raw incoming html emails and headers

### PR DESCRIPTION
**Pull request by McPizza0**
this PR adds functionality to store all incoming email's headers and html body

it then offers a way for users to view the raw email data in the ui

this opens up a modal to view the headers and orginal html body

Raw messages are given an expiry data, after which a cron job will delete the raw data to reduce DB usage

### CODE REVIEW

**Comment by McPizza0** [comment]
* **#186** <a href="https://app.graphite.dev/github/pr/un/inbox/186?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @McPizza0 and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: apps/web-app/components/convos/convoMessageItem.vue [hunk 1]
```diff
@@ -144,5 +174,81 @@
...
+            Raw message will be deleted on
+          </span>
+          <span class="text-sm">{{ rawHtmlData?.rawEmailData.wipeDate }}</span>
+        </div>
+        <div
+          v-if="rawHtmlData?.rawEmailData.html"
+          class="mb-4 flex flex-col gap-2">
+          <span class="text-base-11 text-sm uppercase"> Raw Email </span>
+          <div class="relative h-0 w-full overflow-hidden pb-[60%]">
+            <iframe
```
Comment (BlankParticle) [inline apps/web-app/components/convos/convoMessageItem.vue:242]: add `sandbox=""` to sandbox the iframe

**Review by BlankParticle — CHANGES_REQUESTED** [review]
Everything else looks good

**Review by BlankParticle — APPROVED** [review]

**Comment by McPizza0** [comment]
### Merge activity

* **Mar 31, 8:34 AM EDT**: @McPizza0 merged this pull request with [Graphite](https://app.graphite.dev/github/pr/un/inbox/186).

**Pull request merged**